diff -ru ../../configsets/_default/conf/managed-schema ./managed-schema
--- ../../configsets/_default/conf/managed-schema	2021-06-10 22:23:17.000000000 +0200
+++ ./managed-schema	2021-09-13 23:01:10.391240264 +0200
@@ -121,6 +121,15 @@
     <field name="_nest_path_" type="_nest_path_" /><fieldType name="_nest_path_" class="solr.NestPathField" />
 
     <field name="_text_" type="text_general" indexed="true" stored="false" multiValued="true"/>
+    <field name="lang" type="text_general" indexed="true" stored="true"/>
+    <field name="metadata" type="text_general" indexed="true" stored="true" multiValued="true"/>
+    <field name="cat" type="string" indexed="true" stored="false" multiValued="true"/>
+    <field name="content" type="text_general" indexed="true" stored="true" multiValued="true" termVectors="true" termPositions="true" termOffsets="true"/>
+    <!--field name="content" type="text_general" indexed="true" stored="false" multiValued="true"/-->
+    <!--field name="text" type="text_general" indexed="true" stored="false" multiValued="true"/-->
+
+    <!--copyField source="metadata" dest="text"/>
+    <copyField source="content" dest="text"/-->
 
     <!-- This can be enabled, in case the client does not know what fields may be searched. It isn't enabled by default
          because it's very expensive to index everything twice. -->
diff -ru ../../configsets/_default/conf/solrconfig.xml ./solrconfig.xml
--- ../../configsets/_default/conf/solrconfig.xml	2021-06-10 22:31:39.000000000 +0200
+++ ./solrconfig.xml	2021-09-13 22:56:42.355443771 +0200
@@ -377,7 +377,7 @@
          specified in `solr.xml`, it will have no effect, as that setting also limits the size
          of user specified boolean queries.
       -->
-    <maxBooleanClauses>${solr.max.booleanClauses:1024}</maxBooleanClauses>
+    <maxBooleanClauses>${solr.max.booleanClauses:16384}</maxBooleanClauses>
 
     <!-- Solr Internal Query Caches
 
@@ -838,12 +838,13 @@
       <str name="echoParams">explicit</str>
       <str name="wt">json</str>
       <str name="indent">true</str>
+      <str name="df">content</str>
     </lst>
   </requestHandler>
 
   <initParams path="/update/**,/query,/select,/spell">
     <lst name="defaults">
-      <str name="df">_text_</str>
+      <str name="df">content</str>
     </lst>
   </initParams>
 
